mlos_bench/mlos_bench/tests/tunables/test_tunables_size_props.py

```diff
@@ -81,10 +86,17 @@ def test_tunable_quantized_float_size_props() -> None:
     """Test quantized tunable float size properties."""
     tunable = Tunable(
         name="test",
-        config={"type": "float", "range": [0, 1], "default": 0, "quantization": 0.1},
+        config={
+            "type": "float",
+            "range": [0, 1],
+            "default": 0,
+            "quantization": 10,
+        },
     )
     assert tunable.span == 1
-    assert tunable.cardinality == 11
+    # FIXME:
+    assert tunable.quantization == 10  # == tunable.quantization_num_bins == 11 ?
+    assert tunable.cardinality == 11  # == 10 ?
```

Comment:
@motus I put a few comments in here in my local version and pushed them here

Reply:
yep, thank you! I'll also fix that quantization vs. cardinality issue now.